server/chat/backend/agent/tools/github_repos_tool.py

```
@@ -21,6 +26,14 @@ def get_connected_repos(**kwargs) -> str:
     if not user_id:
         return json.dumps({"error": "No user context available"})
 
+    try:
+        auth = get_any_auth_for_user(user_id)
+    except NoGitHubAuthError:
+        return "GitHub not connected for this user. Install the GitHub App or connect via OAuth."
+    except Exception as e:
+        logger.error(f"Error resolving GitHub auth for user {user_id}: {e}", exc_info=True)
```

Ruff (0.15.12)

[warning] Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)